CODEOWNERS

```diff
@@ -4,5 +4,8 @@
 # Documentation.
 /docs/ @grafana/docs-logs @grafana/loki-team
 
+# Loki operator
+/operator/ @grafana/loki-team @periklis
```

Comment:
Can you please add my co-maintainer having merging rights @xperimental here please.

Reply:
done.